resources/definitions/creality_ender3v3se.def.json

```diff
@@ -37,19 +38,40 @@
         "machine_heated_bed": { "default_value": true },
         "machine_height": { "default_value": 250 },
         "machine_max_acceleration_e": { "value": 5000 },
-        "machine_max_acceleration_x": { "value": 5000.0 },
-        "machine_max_acceleration_y": { "value": 5000.0 },
-        "machine_max_acceleration_z": { "value": 500.0 },
+        "machine_max_acceleration_x": { "value": 2500.0 },
+        "machine_max_acceleration_y": { "value": 2500.0 },
+        "machine_max_acceleration_z": { "value": 2500.0 },
         "machine_max_feedrate_e": { "value": 100 },
         "machine_max_feedrate_x": { "value": 500 },
         "machine_max_feedrate_y": { "value": 500 },
-        "machine_max_feedrate_z": { "value": 30 },
+        "machine_max_feedrate_z": { "value": 50 },
         "machine_name": { "default_value": "Creality Ender-3 V3 SE" },
         "machine_start_gcode": { "default_value": "M220 S100 ;Reset Feedrate\nM221 S100 ;Reset Flowrate\n\nG28 ;Home\n\nM420 S1; Enable mesh leveling\n\nG92 E0 ;Reset Extruder\nG1 Z2.0 F3000 ;Move Z Axis up\nG1 X10.1 Y20 Z0.28 F5000.0 ;Move to start position\nM109 S[material_print_temperature_layer_0]\nG1 X10.1 Y145.0 Z0.28 F1500.0 E15 ;Draw the first line\nG1 X10.4 Y145.0 Z0.28 F5000.0 ;Move to side a little\nG1 X10.4 Y20 Z0.28 F1500.0 E30 ;Draw the second line\nG92 E0  ;Reset Extruder\nG1 E-1.0000 F1800 ;Retract a bit\nG1 Z2.0 F3000 ;Move Z Axis up\nG1 E0.0000 F1800 \n" },
         "machine_width": { "default_value": 220 },
         "retraction_amount": { "value": 0.8 },
         "retraction_speed": { "default_value": 40 },
-        "speed_layer_0": { "value": 30 },
-        "speed_print": { "value": 180 }
+        "speed_layer_0": { "value": 50 },
+        "speed_print": { "value": 60 },
```

Comment:
Why so low? 180 seems to be printing fine for me

Reply:
if print PETG, 180mm too much. that is default params, you always can change it in settings

Reply:
> 180mm too much

What do you mean "too much"?
Ender 3 v3 SE is marketed as able to print 250 mm/s, it should be able to go faster than 60. Sure, we likely want default settings to be conservative, but 60 is too low IMO. Maybe bump it to 100 mm/s?

I did a couple of prints - left is 60 mm/s, right is 180 mm/s. I had to disable minimal layer time and use large test model to make sure speeds are achieved. Prints seem fine:
![ender_3v3se_60_vs_180](https://github.com/Ultimaker/Cura/assets/7951939/3c7a873c-ba67-49fe-8f95-ab97fffef72a)

Problems:

- Horizontal lines on 60 mm/s _seem_ to be caused by the bad flow of the filament as I reuse Ender 3 LACK enclosure without filament feed line
- 180 has slightly worse surface, small horizontal lines that look line under extrusion

Non-default settings: 
Hotend temperature: 230
Initial layer speed: 10 mm/s
Initial layer horizontal expansion: -0.2mm

Reply:
ok. updated speed_print to 100mm/s